cmd/server/main.go

```
@@ -1,25 +1,149 @@
+	"github.com/mguptahub/nanodns/internal/logging"
 	"github.com/mguptahub/nanodns/pkg/config"
 	externaldns "github.com/miekg/dns"
 )
 
+const pidFilePath = "/tmp/nanodns.pid"
+
+var (
+	version          = "dev" // Default version; can be overridden at build time
+	startService     bool
+	stopService      bool
+	installService   bool
```

golangci-lint

var `installService` is unused

(unused)